LoopBack/LBModel.h

```diff
@@ -8,6 +8,68 @@
 #import "SLRemoting.h"
 
 /**
+ * Blocks of this type are executed for a successful method invocation that doesn't return
+ * any value as a callback argument.
+ */
+typedef void (^LBModelVoidSuccessBlock)();
```

Comment:
For convenience (especially for CodeGen), the success block types with supported data types are typedef'ed.